Starforge/Map/Style.cs

```diff
@@ -0,0 +1,70 @@
+﻿using System.Collections.Generic;
+
+namespace Starforge.Map {
+    /// <summary>
+    /// Represents a style element of the level (e.g. a parallax styleground)
+    /// </summary>
+    public class Style : MapElement {
+        public static List<Style> ParseListElement(MapElement root) {
+            List<Style> res = new List<Style>();
+
+            foreach (MapElement style in root.Children) {
+                if (style.Name == "apply") {
+                    foreach (MapElement apply in style.Children) {
+                        res.Add(ParseStyle(apply, style));
+                    }
+                } else {
+                    res.Add(ParseStyle(style, null));
+                }
+            }
+
+            return res;
+        }
+
+        public static Style ParseStyle(MapElement style, MapElement root) {
+            if (root != null && root.Name == "apply") {
+                if (style.Name == "parallax") {
+                    // Apply styleground
+                    return new Parallax(style, root);
+                } else {
```

Comment:
For early error catching and possibly extensibility(? idk) it might be worth to still check the Name in this case too, and if it's neither "parallax" nor "effect" then log it or something. Unless this works differently, then ignore me

Reply:
There is no `effect`. The names of effect styles are simply the ID of the effect (e.g. `windsnow`, `snowfg`, etc). It's a bit weird, but it's how it is.